TanmayeeGujar_HW5-2.ipynb

```diff
@@ -0,0 +1,294 @@
+{
```

Comment:
<p>You are giving your c++ function data types it cannot handle. I think you're lucky that uint and int is the same n. However I'm surprised that the long int works since your function technically cannot handle it.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_PythonCpp/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>